public/scripts/data-util.js

```diff
@@ -154,8 +154,8 @@ var DataUtil = (function() {
         return str;
     }
 
```

Comment:
Can we move this function up visually to live with the other "public" (data-object) methods? :) (not really a big deal!)

Reply:
i like it!  will do that now